Pear/SceneDelegate.swift

```diff
@@ -92,3 +94,47 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
     }
 
 }
+
+extension SceneDelegate: UNUserNotificationCenterDelegate {
+
+    private func setupLocalNotifications() {
+        let center = UNUserNotificationCenter.current()
+        center.requestAuthorization(options: [.alert, .sound]) { (granted, error) in
+            print("granted: \(granted)")
+        }
+        center.delegate = self
+        // get rid of previously scheduled notifications
+        center.removeAllDeliveredNotifications()
+        center.removeAllPendingNotificationRequests()
+        createNotifications(center: center, day: 2, hour: 8)
+        createNotifications(center: center, day: 4, hour: 14)
+        createNotifications(center: center, day: 6, hour: 12)
+    }
+
+    private func createNotifications(center: UNUserNotificationCenter, day: Int, hour: Int) {
+        let content = UNMutableNotificationContent()
+        content.title = "Meet your new pear!"
+        content.body = "Set up this week's chat today 😊"
+        content.sound = UNNotificationSound.default
+        var dateComponents = DateComponents()
+        dateComponents.weekday = day
+        dateComponents.hour = hour
+        dateComponents.minute = 0
+        dateComponents.second = 0
```

Comment:
Couldn't you combine lines 119-123 by doing it all within the constructor.  I think DateComponents(hour: hour, minute: 0, second: 0, weekday: day) would work but I could be wrong.